src/ajax/check_hgvs_dialogue.php

```diff
@@ -0,0 +1,603 @@
+    //  we want each transcript to get a mapping. To get this, we then
+    //  need to call VariantValidator a second time using one of the
+    //  genomic variants as were returned using our first call.
+    if ($sType == 'VOT' && count($aTranscripts) > 1) {
+
+        $aMappedViaGB = (
+        !isset($aMappedVariant['data']['genomic_mappings']['hg38']) ? // Yes=We have a genomic reference from our first call; No=We don't have a genomic reference.
+            array() :
+            $_VV->verifyGenomic($aMappedVariant['data']['genomic_mappings']['hg38'], array(
+                'map_to_transcripts' => true,  // Should we map the variant to transcripts?
+                'predict_protein'    => true,  // Should we get protein predictions?
+                'lift_over'          => false, // Should we get other genomic mappings of this variant?
+                'select_transcripts' => array_diff($aTranscripts, array($sReferenceSequence)), // Which transcripts do we want to map to?
+            ))
+        );
+
+        if (!isset($aMappedViaGB['data']['transcript_mappings'])) {
+            // If for any reason no new transcript mappings were given,
+            //  we cannot perform this extra step, and will thus miss
+            //  some information. We will inform the user.
+            update_dialogue('<br>Your variant could not be mapped to all transcripts due to unknown issues.');
+        }
+
+        $aMappedVariant['data']['transcript_mappings'] = $aMappedViaGB['data']['transcript_mappings'];
+
+        unset($aMappedViaGB); // We don't need the rest of this information.
+    }
+
+    // We have the mapping data and can now send it to the input fields.
+
+    // Save the ['data']['DNA'] variant to the right type of mapping to
+    //  easily add it into the right fields later.
+    if ($sType == 'VOT') {
+        // If the input type was a variant on transcript, the ['data']['DNA']
+        //  field holds information on a transcript mapping.
+        $aMappedVariant['data']['transcript_mappings'][$sReferenceSequence] = $aMappedVariant['data'];
+    } else {
+        // If the input was not a VOT, it was a genomic variant, meaning we
+        //  should add the information into the genomic mappings.
+        $aMappedVariant['data']['genomic_mappings'][$sGenomeBuildID] = $aMappedVariant['data']['DNA'];
+    }
+
+    // Returning the mapping for transcript, RNA and protein variants.
+    foreach ($aMappedVariant['data']['transcript_mappings'] as $sTranscript => $aTranscriptData) {
+        // Filling in the input fields.
+        print('
+        // Adding transcript info to the fields.        
+        var oTranscriptField = $("input").filter(function() { 
+            return $(this).data("id_ncbi") == "' . $sTranscript . '" 
+        });
+        
+        if (!oTranscriptField.prop("disabled")) {
+            oTranscriptField.val("' . $aTranscriptData['DNA'] . '").attr("class", "acc");;
```

Comment:
Earlier, `removeClass()` was used. Is that not used here to make sure other values are overwritten?